### ISSUE TITLE
Increase the font size on the Libki client login screen.

### ISSUE THREAD

**Issue by PPL6265** [Libki/libki-client#74]
Hi,

Is it possible to provide a way to increase the font size on the client login screen? It is currently too small for some clients to see and is an AODA issue.

Thanks.

**Comment by kylemhall** [comment]
I'll see what I can do!

**Comment by kylemhall** [comment]
This should be resolved by Add ability to set Libki client application style sheet #77

This commit allows your to apply styles to any element of the application.

**Comment by fclibtech** [comment]
Hi Kyle,

Would have any sample code? I haven't used QT style sheets before.

Thanks


Kind Regards



Mark Shelton

Library Technology and Innovation Coordinator

Community & Culture

Fraser Coast Regional Council

T  4197 4232 | M 0499 577 616  E  ***@***.******@***.***>


________________________________

**Comment by kylemhall** [comment]
@fclibtech update to the latest master branch of Libki. Go to the system settings page. Find "Client style sheet" under "Advanced settings". Past the following in and adjust the font sizes as needed!

```
QLabel#headerLabel { font-size: 36px }
QLabel#usernameLabel { font-size: 36px }
QLabel#passwordLabel { font-size: 36px }
QLabel#messageLabel { font-size: 36px }
QLabel#reservedLabel { font-size: 36px }
```

**Comment by fclibtech** [comment]
Hi Kyle,

Thanks for that.

Is it possible to increase the text box sizes and fonts for the username and passwords input by the public?
They often have trouble reading what they type in also.


Kind Regards



Mark Shelton

Library Technology and Innovation Coordinator

Community & Culture

Fraser Coast Regional Council

T  4197 4232 | M 0499 577 616  E  ***@***.******@***.***>


________________________________

**Comment by fclibtech** [comment]
Hi Kyle,

Please ignore my earlier email, I think I've worked it out:

QLineEdit#usernameField {font-size: 24px; font-weight: bold }
QLineEdit#passwordField {font-size: 24px; font-weight: bold }

It may be an idea to add some cool examples in the wiki page:

[cid:b9d09c68-5058-4026-97de-106cb64eeb55]


Kind Regards



Mark Shelton

Library Technology and Innovation Coordinator

Community & Culture

Fraser Coast Regional Council

T  4197 4232 | M 0499 577 616  E  ***@***.******@***.***>


________________________________

**Comment by kylemhall** [comment]
@fclibtech as matter of fact I've done just that!

https://github.com/Libki/libki-client/wiki/Styling-the-Libki-client-with-Qt-Style-Sheets

**Comment by fclibtech** [comment]
Fantastic! Thanks Kyle, its working well 😊

Kind Regards

Mark Shelton
Library Technology and Innovation Coordinator
Community & Culture
Fraser Coast Regional Council
T  4197 4232 | M 0499 577 616 | E  ***@***.******@***.***>

[TRAITS Logo]
Disclaimer: If you receive this email by mistake please notify the sender and do not make any use of it. Council does not waive any privilege, confidentiality or copyright associated with this email or any attachments.
